contracts/Market.sol

```diff
@@ -1078,6 +1079,8 @@ contract Market is
         certificateAmount: amount,
         supplier: supplier
       });
+    uint256 currentPrice = _priceMultiple;
+    _priceMultiple = customPriceMultiple;
```

Comment:
I think we should move this to the top of the function to reduce future risk of the prior functions introducing changes that rely on this value

Reply:
good idea